_stbt/core.py

```diff
@@ -1741,7 +1742,7 @@ def _match(image, template, match_parameters, imglog):
     region = Region(position.x, position.y,
                     template.shape[1], template.shape[0])
 
-    return matched, region, first_pass_certainty
+    return matched, list(region), first_pass_certainty
```

Comment:
I suppose it's too inconvenient to teach the json encoder how to serialize/deserialize `Region` objects... anyway if we expose the caching mechanism to user-defined image-processing functions then they wouldn't be able to extend the json encoder/decoder either... so I suppose it makes sense to stick to the default objects support by  `json.JSONEncoder`.